### PR TITLE
Update RotationCheck.java

### PR DESCRIPTION
**Pull request by optsri**
I hope that what I have already done is OK.
waiting for your comments.

### CODE REVIEW

File: src/ameedf/jb8223/assignments/RotationCheck.java [hunk 1]
```diff
@@ -43,5 +43,89 @@
 public class RotationCheck {
     public static void main(String[] args) {
         // TODO: enter your code here :)
+    Scanner input = new Scanner(System.in);
+		int maxvalue = 1_000_000_000;
```
Comment (ameedf) [inline src/ameedf/jb8223/assignments/RotationCheck.java:47]: Follow the camelCase best practice: maxValue

File: src/ameedf/jb8223/assignments/RotationCheck.java [hunk 2]
```diff
@@ -43,5 +43,89 @@
...
+			LengthTemp1--;
+			Num1 /= 10;
+		}
+		// Create an Array for Number#2 + putting each digit in the array#2
+		int LengthTemp2 = LengthNum2;
+		int DigitsNum2 = 0;
+		int[] arraynum2 = new int[LengthNum2];
+		for (int i = 0; i < LengthNum2; i++) {
+			DigitsNum2 = Num2 % 10;
+			arraynum2[LengthTemp2 - 1] = DigitsNum2;
```
Comment (ameedf) [inline src/ameedf/jb8223/assignments/RotationCheck.java:110]: did you mean arraynum2[i] = DigitsNum2 ??
remember you need to populate all the array, so you have to go by the index (i)

File: src/ameedf/jb8223/assignments/RotationCheck.java [hunk 3]
```diff
@@ -43,5 +43,89 @@
...
+			System.out.println("The length of the numbet isn't equal - don't need to rotate & check the numbers !");
+		}
+		// Checking if the number are equal after rotating
+		else {
+			while (Num1 != Num2) {
+				for (int i = 0; i < LengthNum1 - 1; i++) {
+					int LengthNum1Temp = LengthNum1;
+					arraynum1[LengthNum1Temp - 1] = arraynum1[i];
+					LengthNum1Temp--;
+		// I Got Stuck !
```
Comment (ameedf) [inline src/ameedf/jb8223/assignments/RotationCheck.java:125]: The question is: what are trying to do here ? rotate ?

**Comment by gridan1982** [comment]
import java.util.Scanner;

public class RotationCheck {
	

	public static void main(String []arg) {
		int MaximumNumberAllowed = 1000000000;
		System.out.println(" Enter two different numbers in the range of 0 - "+ MaximumNumberAllowed );
		
		System.out.println("Enter the first number");
		Scanner input = new Scanner(System.in);
		int FirstNum = input.nextInt(); 
		
		while (FirstNum < 0 || FirstNum > MaximumNumberAllowed ){
			System.out.println("The number is not in range, enter the number agian");
		FirstNum = input.nextInt();}
		
		
		System.out.println("Enter the second number");
		Scanner input2 =new Scanner(System.in);
		int SecondNum = input2.nextInt();
		
		while (SecondNum < 0 || SecondNum > MaximumNumberAllowed ){
			System.out.println("The number is not in range, enter the number agian");
		SecondNum = input2.nextInt();
		}
		while (FirstNum == SecondNum) {
			System.out.println("The two numbers can't be identical, try agian ");
					FirstNum = input.nextInt();
					SecondNum =input2.nextInt();
		
		}
		

		 int[] firstdigits;
	        if (FirstNum == 0) {
	            firstdigits = new int[1]; 
	        } else {
	            int FirstNumLenght = 0;
	            int temp = FirstNum;
	            while (temp != 0) {
	                temp = temp / 10;
	                FirstNumLenght++;
	            }
	        
	        firstdigits = new int[FirstNumLenght];
            temp = FirstNum;
            for (int i = 0; i < FirstNumLenght; i++) {
                firstdigits[i] = (temp % 10);
                temp /= 10;
            
            }
            
                int[]SecondDigits;
  	    	  if (SecondNum ==0 ) {
  	    		  SecondDigits = new int[1];
  	    	  }else { 
  	    		  int SecondNumLength = 0;
  	    		  int temp1 = SecondNum;
  	    		  while (temp1 != 0) {
  	    		  temp1 = temp1/ 10;
  	    		  SecondNumLength++;
  	    		  
  	    		  if (SecondNumLength != FirstNumLenght) {
  	    			System.out.println("The two number should have the same amount of digits. re enter the numbers");  
  	    			FirstNum = input.nextInt();// the input is with out the  restrictions from the top, can i do it in class?
					SecondNum =input2.nextInt();
  	    		  }
  	    		  else { 
  	    			  
  	    		  
					int rotation = 0;
					int tmplastnum = firstdigits[FirstNumLenght-1]; 
				while (firstdigits != SecondDigits ) {
					for  (int i = FirstNumLenght-1; i >0 ;i--) {
						firstdigits[i]=firstdigits[i-1];
						rotation++;
						firstdigits[0]=tmplastnum;
					
					}
					
  	    		  
				 // ihave got stack here
					
					
					
					
					
					
			
					
						
  	    		  
  	    		  }
  	    		  }   
  	    		   } 
            
	        }
	
	
	
	
	}
}

**Pull request closed without merging**